pkg/repository/v1_repository.go

```diff
@@ -176,11 +181,17 @@ func (r *V1Repository) selectVersion(id string, versions map[string]v1manifest.V
 		for v := range versions {
 			versionSlice = append(versionSlice, v)
 		}
-		sort.Strings(versionSlice)
-		// TODO I think it is not actually correct semver order because versions can have a text suffix.
-		version := versionSlice[len(versionSlice)-1]
-		item := versions[version]
-		return version, &item, nil
+
+		var latest string
+		var latestItem v1manifest.VersionItem
+		for version, item := range versions {
+			if latest == "" || semver.Compare("v"+version, "v"+latest) > 0 {
```

Comment:
the current generate `version` has include the "v" actually

Reply:
I don't understand, versions should come straight from the manifest and therefore not have the "v". Where is it added?

Reply:
the origin manifest already have the "v" generated by `tools/migrate`

Reply:
I change this line to remove the "v" and fix the related tests, others look good. I will merge this first.
If we want to use it as without the "v", we should change it both later.